### ISSUE TITLE
Aqara H1 Hub New Firmware 4.0.5_0009.0746

### ISSUE THREAD

**Issue by miniknife88** [niceboygithub/AqaraM1SM2fw#86]
https://cdn.aqara.com/cdn/opencloud-product/mainland/product-firmware/prd/lumi.gateway.sacn01/20230906143034_lumi.gateway.sacn01_AIOT_V4.0.5_0009.0746_20230905_fcfab3.bin

**Comment by almirus** [comment]
@miniknife88 please help me
how to get firmware file for aqara g2h (EU)

**Comment by ashottonoyan** [comment]
The update doesn't work:

```
# cd /tmp && wget -O /tmp/curl "http://master.dl.sourceforge.net/project/mgl03/bin/curl?viasf=1" && chmod a+x /tmp/curl
-s -k -L -o /tmp/h1_update.sh https://raw.githubusercontent.com/niceboygithub/AqaraM1SM2fw/main/modified/H1/h1_update.sh
chmod a+x /tmp/h1_update.sh && /tmp/h1_update.shConnecting to master.dl.sourceforge.net (216.105.38.12:80)
curl                 100% |*************************************************************************************************************************|  2271k  0:00:00 ETA
# /tmp/curl -s -k -L -o /tmp/h1_update.sh https://raw.githubusercontent.com/niceboygithub/AqaraM1SM2fw/main/modified/H1/h1_update.sh
# chmod a+x /tmp/h1_update.sh && /tmp/h1_update.sh
h1_update.sh revision: 2
type: lumi.gateway.sacn01, model: AH_BOX
platform: aiot, path: /tmp/fw.tar.gz, sign: 0
Unpack path : /tmp/ota_dir
Firmware path : /data/ota_dir
Available ROM size(b): 34680832
Available RAM size(b): 2310144
Update to 4.0.5_0009.0746
Get packages, please wait...
Update failed, reason: getpack failed!
# 
```

**Comment by niceboygithub** [comment]
Please try again.

**Comment by ashottonoyan** [comment]
Extremely slow and getting this `fw_update /tmp/ota_dir/root.bin failed`


```
# cd /tmp && wget -O /tmp/curl "http://master.dl.sourceforge.net/project/mgl03/bin/curl?viasf=1" && chmod a+x /tmp/curl
-s -k -L -o /tmp/h1_update.sh https://raw.githubusercontent.com/niceboygithub/AqaraM1SM2fw/main/modified/H1/h1_update.sh
chmod a+x /tmp/h1_update.sh && /tmp/h1_update.shConnecting to master.dl.sourceforge.net (216.105.38.12:80)
curl                 100% |************************************************************************************|  2271k  0:00:00 ETA
# /tmp/curl -s -k -L -o /tmp/h1_update.sh https://raw.githubusercontent.com/niceboygithub/AqaraM1SM2fw/main/modified/H1/h1_update.sh
# chmod a+x /tmp/h1_update.sh && /tmp/h1_update.sh
h1_update.sh revision: 2
type: lumi.gateway.sacn01, model: AH_BOX
platform: aiot, path: /tmp/fw.tar.gz, sign: 0
Unpack path : /tmp/ota_dir
Firmware path : /data/ota_dir
Available ROM size(b): 34611200
Available RAM size(b): 20578304
Update to 4.0.5_0009.0746
Get packages, please wait...
Got packages done
fw_update /tmp/ota_dir/linux.bin suceess
fw_update /tmp/ota_dir/root.bin failed

Update Done, Please manually reboot!
# 
```

**Comment by miniknife88** [comment]
Update failed，rootfs_4.0.5_0009.0746_modified.bin
Error message：
Err: Unsupport sign:hsqs
